code/game/objects/effects/decals/turfdecal/tilecoloring.dm

```diff
@@ -60,6 +72,14 @@
 /obj/effect/turf_decal/trimline/Initialize()
 	if(SSevents.holidays && SSevents.holidays[APRIL_FOOLS])
 		color = "#[random_short_color()]"
+	if(SSevents.holidays && SSevents.holidays[PRIDE_MONTH])
```

Comment:
i would move these to be a subtype of a shared path but that would result in fucktons of path conflicts